Sources/Target/Windows/Process.cpp

```diff
@@ -590,13 +589,10 @@ ErrorCode Process::enumerateSharedLibraries(
       return Platform::TranslateError();
     sl.path = ds2::Utils::WideToNarrowString(std::wstring(nameStr, nameSize));
 
-    // The following two transforms ensure that the paths we return to the
+    // The following transform ensures that the paths we return to the
     // debugger look like unix paths. This shouldn't be required but LLDB seems
     // to be having trouble with paths when the host and the remote don't use
     // the same path separator.
```

Comment:
I don't think that we are giving unix like paths here ...

```suggestion
    // Normalize paths to `/` as the arc separator. This shouldn't be required
    // but LLDB seems to have trouble with paths when the host and the remote
    // don't use the same path separator.
```

Reply:
Done.